firstvoices/backend/management/commands/convert_lyrics_draftjs_to_text.py

```diff
@@ -0,0 +1,44 @@
+import json
+from json import JSONDecodeError
+
+from django.core.management import BaseCommand
+
+from backend.models import Lyric
+
+delimiter = "\n"
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        lyric_draftjs_to_text()
+
+
+def lyric_draftjs_to_text():
+    lyric_text_list = Lyric.objects.exclude(text__exact="")
+    lyric_translation_list = Lyric.objects.exclude(translation__exact="")
+
+    for lyric in lyric_text_list:
+        try:
+            new_lyric_text = extract_plain_text(json.loads(lyric.text))
+            lyric.text = new_lyric_text
+            lyric.save()
+        except JSONDecodeError:
+            continue
+
+    for lyric in lyric_translation_list:
+        try:
+            new_lyric_translation = extract_plain_text(json.loads(lyric.translation))
+            lyric.translation = new_lyric_translation
+            lyric.save()
+        except JSONDecodeError:
+            continue
+
+
+def extract_plain_text(draftjs):
+    plain_text = ""
```

Comment:
This is totally fine and looks correct to me. Just as a note for interest, there are some built-in python features for doing this-- you'd get the same result from `delimiter.join([ block["text"] for block in draftjs["blocks"] ])`